scout/adapter/mongo/variant.py

```diff
@@ -689,8 +699,13 @@ def get_region_vcf(
             for header_line in vcf_obj.raw_header.split("\n"):
                 if len(header_line) > 3:
                     temp.write(header_line + "\n")
-            for variant in vcf_obj(region):
-                temp.write(str(variant))
+            try:
+                for variant in vcf_obj(region):
+                    temp.write(str(variant))
+            except Exception:
+                raise FileNotFoundError(
+                    "Could not find index for {}".format(variant_file)
```

Comment:
mmm I don't understand, why it's the index file that is not found here?

Reply:
Because we are accessing variants from the vcf object with a region. If there is not index cyvcf2 will raise an exception